frontend/src/app/shared/community-agreement/community-agreement.widget.css

```diff
@@ -15,6 +15,6 @@
 
 ::-webkit-scrollbar-thumb {
   border-radius: 4px;
-  background-color: rgba(0, 0, 0, .5);
-  box-shadow: 0 0 1px rgba(255, 255, 255, .5);
-}
\ No newline at end of file
+  background-color: rgba(0, 0, 0, 0.5);
+  box-shadow: 0 0 1px rgba(255, 255, 255, 0.5);
```

Comment:
I think this code is from the legacy implementation (in #248), however this overrides the new style you added - consider deleting!

An important consideration, though -- dialogs have a different color than panes / cards do, so the thumb color applied globally may not look the best here. You can consider adding a custom color to all dialogs in `styles.scss` using something like the following:

```css
/* Regular styling */
::-webkit-scrollbar-track {
  background-color: ...
}

/* Dialog styling */
.mat-mdc-dialog-content {
  ::-webkit-scrollbar-track {
    background-color: ...
  }
}

```